README.md

```diff
@@ -4,6 +4,13 @@
 
 [![Build Status](https://semaphoreci.com/api/v1/joshsmith/code-corps-phoenix/branches/add-users-endpoint/badge.svg)](https://semaphoreci.com/joshsmith/code-corps-phoenix)
 
+
+## Installing with Docker
+
+To make your life easier, you can just clone this repository and use our Docker container. [Follow this guide to get started.](docs/INSTALLING.md)
+
+## Running the app without Docker
```

Comment:
We might want to remove this section altogether. I would rather not encourage people to run the app without Docker.